go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/spf13/pflag v1.0.10
 	github.com/stretchr/testify v1.11.1
 	gopkg.in/yaml.v2 v2.4.0
+	helm.sh/helm/v3 v3.16.1
```

Comment:
Im a little concerned about pulling this in which then pulls in a bunch of its own deps.
Is there something lighter weight that we can do to avoid Helm just to list releases? Could we for example check of a Deployment or a Helm Secret?